Sources/Splash/Syntax/SyntaxHighlighter.swift

```diff
@@ -35,8 +35,9 @@ public struct SyntaxHighlighter<Format: OutputFormat> {
                 state = (token, type)
                 return
             }
-
-            builder.addToken(token, ofType: type)
+            if token != whitespace {
```

Comment:
This works nicely if there is a single whitespace character prefixing the code, but fails if there is more than one whitespace character.

This works for me:
`if !whitespace.contains(token) {`